src/types/types.ts

```diff
@@ -308,3 +308,30 @@ export interface TDictionary<T> {
 
 export type TOrganisationDictionary = TDictionary<TOrganisation>;
 export type TCorpusTypeDictionary = TDictionary<TCorpusType>;
+
+export type Concept = {
```

Comment:
Suggestion: we use the `TConcept` prefix with T for consistency (open to changing this if people feel strongly opinionated on this - just something I've carried over from previous projects)